src/modules/endpoint/types.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import moment from 'moment'
+
+export type AvailableLanguages = {
+  de?: string,
+  en?: string,
+  ar?: string,
+  fa?: string,
+  fr?: string,
+  tr?: string,
+  ku?: string,
+  ru?: string,
+  ti?: string,
+  sr?: string,
+  am?: string,
+  per?: string,
+  pl?: string
+}
+
+export type EventType = {
```

Comment:
I think the models should stay classes. I'm not that sure what the real difference is between a Type Alias and a Class Type. Maybe our models will have (or had) methods which are not possible to add with flow types.

None the less each type/class should belong to a separate file. Also we should name this Event or EventModel. You would not name it an EventClass.